scripts/lametro/fast-full-bill-scrape.sh

```diff
@@ -0,0 +1,6 @@
+#/bin/sh
+
+(cd $APPDIR && \
```

Comment:
since you are in a script you could 

do it like

```bash
set -e # exit on first error
cd $APPDIR
$PUPADIR update --datadir=/cache/bills/_data/ lametro --scrape bills --window=0 --rpm=0
$PUPADIR update --datadir=/cache/bills/_data/ lametro --import
```